.github/workflows/build-pdb.yml

```diff
@@ -30,3 +30,51 @@ jobs:
       
     - name: Build with Maven
       run: mvn -B clean install --file pom.xml -DskipTests
+
+  build-jdk11:
+
+    runs-on: ubuntu-latest
```

Comment:
I think we should force ubuntu 18 for now. WDYT?

Reply:
Why can't we just let this run on latest?

Reply:
It was just a suggestion, because the latest can change any time by github when a new release of ubuntu is released.
But yeah, I guess it doesn't matter much.